### ISSUE TITLE
Trying to use two different ics sources

### ISSUE THREAD

**Issue by luebbe** [mampfes/hacs_waste_collection_schedule#470]
Hi folks,

I'm trying to get two different ics sources into my schedule. The source configuration for both seems to be ok, but I can't get the sensor configuration to work. My goal is to have one "final" sensor that combines the different sources into one, so I can have only one card for the waste collection on the UI.

configuration.yaml contains mainly aliases to nicer names and icon definitions which I left out here.

```yaml
waste_collection_schedule:
  sources:
    - name: ics
      args:
        url: !secret waste_url
      calendar_title: Müllabfuhr
      customize:
        - type: Restabfallbehaelter
          alias: Restmülltonne
        - type: Wertstoffbehaelter
          alias: Wertstofftonne
        - type: Biobehaelter
          alias: Biotonne
    - name: ics
      args:
        url: !secret paper_url
      calendar_title: Altpapier
      customize:
        - type: Leerung der Blauen Tonne Schlauen Tonne
          alias: Altpapier
```

from sensors.yaml

```yaml
- platform: waste_collection_schedule
  name: Müllabfuhr
  value_template: '{{value.types|join(", ")}}|{{value.daysTo}}'
  types:
    - Biotonne
    - Restmülltonne
    - Wertstofftonne
    - Altpapier   # I didn't really expect this to work

- platform: waste_collection_schedule
  name: Altpapier
  value_template: '{{value.types|join(", ")}}|{{value.daysTo}}'
  types:
    - Altpapier  # I would have expected this to work
```

![grafik](https://user-images.githubusercontent.com/3872576/207952510-470cb3f7-ac3e-448d-b449-0c098e6761db.png)
The "Müllabfuhr" sensor works for the three first waste types. It does so for over a year. I added "Altpapier", but didn't really expect this to work.

![grafik](https://user-images.githubusercontent.com/3872576/207952807-e922a0ed-8a8f-48d3-93e5-a9107e1476d2.png)
The "Altpapier" sensor shows "unknown"

Both calendars created from the sources in "configuration.yaml" are shown on the ui and contain data. The "Altpapier" calendar shows the next collection date.

Why is the status of the "Altpapier" sensor unknown? Is it because the upcoming collection is next year? 
The format of the dates in the two ICS files is slightly different, but since they show up in the calendars, I assume that parsing the date is not the problem.
I can provide the ICS files if this is of help.

**Comment by dt215git** [comment]
You have two sources, you may need to specify which source the sensors are using.
The [source_index](https://github.com/mampfes/hacs_waste_collection_schedule#configuration-variables-1) config might fix the issue.

**Comment by luebbe** [comment]
Very cool @dt215git that was the hint that I needed. Now I have two working sensors. The next step for me is to figure out how to combine these two sensors into one, so I can base the same card and automation to send notifications on it.

**Comment by mampfes** [comment]
Merging the data of multiple sources into one sensor is not supported yet. But I'm working on it. Stay tuned.

**Comment by mampfes** [comment]
fixed by #477
